### ISSUE TITLE
Doesn't work in Edge

### ISSUE THREAD

**Issue by andrecalvo** [boopathi/react-svg-loader#289]
When using the loader like so to load an inline SVG...
`import AlertIcon from "-!react-svg-loader!images/foo.svg;`

Edge errors as it can't understand the rest/spread syntax.

**Comment by tf** [comment]
Using the rollup variant of this loader, I needed to configure the babel plugin to also compile files with `svg` extension to make it work:

```js
plugins: [
  babel({
    exclude: 'node_modules/**',
    extensions: ['js', 'jsx', 'svg']
  }),
  resolve(),
  commonjs(),
  reactSvg({
    svgo: {multipass: true}
  })
]
```

**Comment by LucaMele** [comment]
Agree. The build export should be a traditional ESM (ES5 + import/export).
